etc/kayobe/ansible/get-cloud-facts.yml

```
@@ -0,0 +1,124 @@
+---
+- name: Get facts for control host
```

nit: `Set facts` rather than `Get facts`
```suggestion
- name: Set facts for control host
```
Applies to subsequent plays as well